components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -4432,13 +4450,16 @@ func TestCreateProject(t *testing.T) {
 				Type:   storage.ChefManaged,
 				Status: storage.NoRules.String(),
 			}
-			resp, err := store.CreateProject(ctx, &chefManagedProject)
+			resp, err := store.CreateProject(ctx, &chefManagedProject, false)
 			require.NoError(t, err)
 			require.Equal(t, &chefManagedProject, resp)
 		},
 	}
 
 	for name, test := range cases {
+		insertTestRole(t, db, v2.ViewerRoleID, "viewer", []string{"any"}, []string{})
+		insertTestRole(t, db, v2.EditorRoleID, "editor", []string{"any"}, []string{})
+		insertTestRole(t, db, v2.ProjectOwnerRoleID, "project owner", []string{"any"}, []string{})
```

Comment:
Again, tests for creating projects now have to have the default roles available.